community/surfraw/build

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh -e
+
+./configure \
+	--prefix=/usr \
+	--sysconfdir=/etc
+
+make
+
+# For some reason, surfraw's Makefile
+# gives an error when those directories
+# do not exist (even though the Makefile
+# creates it before exiting with error).
+
+for dir in lib share bin ; do
+	mkdir -p "$1/usr/$dir"
+done
+mkdir -p "$1/etc"
```

Comment:
This might be something we're able to fix in a more elegant way. I'll look into it.

Reply:
Yup! Setting `MKDIR_P` to run `install` without `-c` fixes the issue.

```sh
#!/bin/sh -e

export MKDIR_P="install -d"

./configure \
    --prefix=/usr \
    --sysconfdir=/etc

make
make DESTDIR="$1" install
```